src/mcts/search.cc

```diff
@@ -1145,6 +1145,22 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
       }
       return NodeToProcess::Collision(node, depth, collision_limit);
     }
+    // Probably best place to check for two-fold draws consistently.
+    // Depth starts with 1 at root, so real depth is depth - 1
+    if (node->IsTwofoldTerminal() && ( params_.GetTwoFoldDrawLevel() > 0 ) ) {
```

Comment:
nit: parens around `>` expression aren't necessary and clang-format whitespace

Reply:
Changed (I hope).